query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/commons/LdapServiceCommons.java

```diff
@@ -226,21 +253,41 @@ private LdapConfigurationField ldapLoginServiceToLdapConfiguration(Map<String, O
             .loginUserAttribute(mapValue(props, LdapLoginServiceProperties.LOGIN_USER_ATTRIBUTE))
             .memberAttributeReferencedInGroup(
                 mapValue(props, LdapLoginServiceProperties.MEMBERSHIP_USER_ATTRIBUTE))
+            .groupAttributeHoldingMember(
+                mapValue(props, LdapLoginServiceProperties.MEMBER_NAME_ATTRIBUTE))
             .baseUserDn(mapValue(props, LdapLoginServiceProperties.USER_BASE_DN))
             .baseGroupDn(mapValue(props, LdapLoginServiceProperties.GROUP_BASE_DN))
             .useCase(AUTHENTICATION);
 
     return new LdapConfigurationField()
         .connection(connection)
+        .loadBalancing(loadBalancing)
         .bindUserInfo(bindUserInfo)
         .settings(settings)
         .pid(mapValue(props, SERVICE_PID_KEY));
   }
 
-  private LdapConnectionField getLdapConnectionField(
+  private ListField<LdapConnectionField> getLdapConnectionField(
       Map<String, Object> props, String ldapUrl, String startTls) {
+    ListField<LdapConnectionField> connection = new LdapConnectionField.ListImpl();
+
+    Boolean startTlsB = (Boolean) props.get(startTls);
```

Comment:
✏️ I rarely complain about variable naming, but this variation on Hungarian Notation is...not the best. It would be less painful (not a ton, but a bit) if the parameter were renamed to `startTlsProp` or `startTlsKey`; it would be less painful still to rename this variable to something like `isStartTls` to at least encode its semantics instead of its type.